api/controller/payments.js

```diff
@@ -0,0 +1,54 @@
+class PaymentController {
+  constructor ({ paymentsModel, userModel, courseModel }) {
+    this.paymentsModel = paymentsModel
+    this.userModel = userModel
+    this.courseModel = courseModel
+  }
+
+  async stripeWebhook (req, res) {
+    const sig = req.headers['stripe-signature']
+    const response = await this.payments.verifyStripeWebhook(sig, req.rawBody)
+    req.log.info(`stripe verify response:  ${JSON.stringify(response)}`)
+
+    return response
```

Comment:
this.payments es paymentsModel o reconoce la clase de payments

Reply:
Es el model, gracias!